docs/clusterdefinition.md

```diff
@@ -53,6 +53,7 @@ Here are the valid values for the orchestrator types:
 | gcLowThreshold                  | no       | Sets the --image-gc-low-threshold value on the kublet configuration. Default is 80. [See kubelet Garbage Collection](https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/)                                                                                                                                                                                                  |
 | kubeletConfig                   | no       | Configure various runtime configuration for kubelet. See `kubeletConfig` [below](#feat-kubelet-config)                                                                                                                                                                                                                                                                                                        |
 | kubernetesImageBase             | no       | Specifies the base URL (everything preceding the actual image filename) of the kubernetes hyperkube image to use for cluster deployment, e.g., `k8s.gcr.io/`                                                                                                                                                                                                                                     |
+| loadBalancerSku                 | no       | Sku of Load Balancer and Public IP. Candidate values are: `basic` and `standard`. If not set, it will be default to basic. Requires Kubernetes 1.11 or newer.                                                                                                                                                                                                                                                                                                        |
```

Comment:
Please add documentation for LB outbound rule workaround + what is expected in v1.12